content/200-concepts/100-components/01-prisma-schema/07-referential-actions.mdx

```diff
@@ -0,0 +1,327 @@
+---
+title: 'Referential actions'
+metaTitle: 'Referential actions'
+metaDescription: 'Referential actions let you define the update and delete behavior of related models on the database level'
+tocDepth: 2
+preview: true
+---
+
+<TopBlock>
+
+As of **2.26.0** Prisma now supports referential actions as part of the Prisma Schema Language.
```

Comment:
```suggestion
As of [2.26.0](https://github.com/prisma/prisma/releases/tag/2.26.0] Prisma now supports _referential actions_ as part of the Prisma Schema Language.
```

Reply:
I think we should add the link to the release notes and explicitly mentioning _early_ that this is a preview feature is necessary to set expectations.